resources/views/components/exhibition-card.blade.php

```diff
@@ -49,8 +49,11 @@
                     <p class="text-dark">Included in General admission</p> --}}
                 @endif
                 <p class="text-dark">
-                    {{  Carbon\Carbon::parse($startDate)->format('j F Y') }} to
-                    {{  Carbon\Carbon::parse($endDate)->format('j F Y') }}
+                    {{  Carbon\Carbon::parse($startDate)->format('j F Y') }}
+                    @unless($displayEndDate)
+                    to
+                    {{ Carbon\Carbon::parse($endDate)->format('j F Y') }}
+                @endunless
```

Comment:
Indentation looks off here? But this might just be Github being weird (if it isn't indented on your local, please correct it, otherwise this is OK to leave)